proxy/handler_test.go

```diff
@@ -46,15 +48,16 @@ func Test_NewHandlerFunc_NoPanic(t *testing.T) {
 		}
 	}()
 
-	proxyFunc := NewHandlerFunc(time.Second, &testBaseURLResolver{})
+	config := types.FaaSConfig{ReadTimeout: time.Second}
+	proxyFunc := NewHandlerFunc(config, &testBaseURLResolver{})
 	if proxyFunc == nil {
 		t.Errorf("proxy handler func is nil")
 	}
 }
 
 func Test_ProxyHandler_NonAllowedMethods(t *testing.T) {
-
-	proxyFunc := NewHandlerFunc(time.Second, &testBaseURLResolver{})
+	config := types.FaaSConfig{ReadTimeout: time.Second}
```

Comment:
A nit, but can we do `time.Second * 1` to be more explicit? Would `time.Millisecond * 100` be OK too?

Reply:
sure

Reply:
I updated it to `100 * time.Millisecond`